scico/test/linop/xray/test_astra.py

```diff
@@ -30,6 +29,17 @@
 RTOL_GPU_RANDOM_INPUT = 1.0
 
 
+def make_im(Nx, Ny, is_3d=True):
+    x, y = snp.meshgrid(snp.linspace(-1, 1, Nx), snp.linspace(-1, 1, Ny))
```

Comment:
(minor and old bug): The shape of `x` after this line is `(Ny, Nx)` which is inconsistent with how that function is later used (`x = make_im(A.input_shape[0], A.input_shape[1], is_3d=False)`). A quick fix would be ` x, y = snp.meshgrid(snp.linspace(-1, 1, Nx), snp.linspace(-1, 1, Ny), indexing='ij')`.

Reply:
Fixed (also in `test_svmbir`).